velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -696,6 +696,56 @@ TEST_F(DateTimeFunctionsTest, hour) {
   EXPECT_EQ(2, hour("1969-01-01 13:23:00.001"));
 }
 
+TEST_F(DateTimeFunctionsTest, minute) {
+  const auto minute = [&](const StringView timestampStr) {
+    const auto timeStamp =
+        std::make_optional(util::fromTimestampString(timestampStr));
+    return evaluateOnce<int32_t>("minute(c0)", timeStamp);
+  };
+
+  EXPECT_EQ(23, minute("2024-01-08 00:23:00.001"));
```

Comment:
Would you use the same timestamps, but different time zones to illustrate the differences in results? I expect no difference for timezones that are full-hours offset from UTC and 30-min difference for timezones that are offset by half-hours.

Reply:
I've used 4 test cases and illustrated results with full-hour offset and 30-min offset time zones.